### PR TITLE
Small syntax updates to Architecture and RNs

### PR DESCRIPTION
**Pull request by melcorr**
To bring into line with suggestions in #1949

### CODE REVIEW

File: _includes/manuals/nightly/1.2_release_notes.md [hunk 1]
```diff
@@ -22,16 +22,16 @@ For more information see the [RFC](https://community.theforeman.org/t/drop-requi
...
+Note that this support statement refers to running Foreman and Foreman Smart Proxy themselves on EL7. Managing EL7 client systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
 
 #### Running Foreman on Debian 10
 
 Now that Debian 11 is supported, Debian 10 support is deprecated and will be dropped with Foreman 3.4
 
-Note that this is only support to run Foreman and Foreman Proxy themselves on Debian 10. Managing Debia 10 systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
+Note that this support statement refers to running Foreman and Foreman Proxy themselves on Debian 10. Managing Debian 10 client systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
```
Comment (ekohl) [inline _includes/manuals/nightly/1.2_release_notes.md]: I'm slightly ashamed I had a typo in this.

File: _includes/manuals/nightly/1.2_release_notes.md [hunk 2]
```diff
@@ -22,16 +22,16 @@ For more information see the [RFC](https://community.theforeman.org/t/drop-requi
 
 Foreman 2.1 introduced EL8 support and Katello 4.0 (on Foreman 2.4) followed.
 While it's currently undecided when EL7 support will be dropped exactly, this is an early notice given the considerable number of EL7 deployments.
-For fresh installations, EL8 should considered the preferred target.
+For fresh installations, it is advisable to install on EL8.
 Existing installations should start thinking about a migration plan.
 
-Note that this is only support to run Foreman and Foreman Proxy themselves on EL7. Managing EL7 systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
+Note that this support statement refers to running Foreman and Foreman Smart Proxy themselves on EL7. Managing EL7 client systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
```
Comment (ekohl) [inline _includes/manuals/nightly/1.2_release_notes.md]: I see your point about this but vanilla Foreman doesn't really have clients. It just has hosts. Those hosts may run servers so users may consider them server systems. Should we rephrase this to:
```suggestion
Note that this support statement refers to running Foreman and Foreman Smart Proxy themselves on EL7. Managing EL7 hosts using Foreman remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
```

File: _includes/manuals/nightly/1.1_architecture.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
 
 ## Foreman Architecture
 
-A Foreman installation will always contain a central foreman instance
+A Foreman installation will always contain a central Foreman instance
```
Comment (ekohl) [inline _includes/manuals/nightly/1.1_architecture.md:4]: This has been unchanged for such a long time that I'm wondering about cherry picks. Right now I'm leaning to all supported versions and ignore < 3.0. Thoughts?

**Review by ekohl — APPROVED** [review]
Thanks!

File: _includes/manuals/nightly/1.2_release_notes.md (hunk 2 above)
Comment (melcorr) [reply _includes/manuals/nightly/1.2_release_notes.md]: I use host/client interchangeably. I found system unclear so I tried to clarify. I'd rather leave the "using Foreman" part and stick with hosts or host systems.

Comment (ekohl) [reply _includes/manuals/nightly/1.2_release_notes.md]: I'm also fine with leaving out "using Foreman". I was recently complaining about the ambiguity problems with these terms. They're so overloaded. So let's go with this?
```suggestion
Note that this support statement refers to running Foreman and Foreman Smart Proxy themselves on EL7. Managing EL7 hosts remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
```

On a related note, I looked at the common uses of client vs host in the manual and every time you do that, you find an issue. Git tells me I'm to blame for this one: https://github.com/theforeman/theforeman.org/pull/1954.

**Review by ekohl — APPROVED** [review]
Thanks!

File: _includes/manuals/nightly/1.2_release_notes.md (hunk 2 above)
Comment (melcorr) [reply _includes/manuals/nightly/1.2_release_notes.md]: A definition list in the planning guide over on docs.theforeman.org would be great. I've tried to curb random vague usage. I held up the publication of ATIX's Foreman ACD guide for days trying to get "service" taken out but I had no better offerings for them and no one else had either, so I had to let it pass. I'd prefer neologisms that exist only for Foreman and clearly defined than "Service/system/host" all used to mean something similar.

Comment (ekohl) [reply _includes/manuals/nightly/1.2_release_notes.md]: I completely agree. It's confusing for users to read one thing in the docs and then see another thing in the UI. It may even be its own guide that all other guides could refer to.

On a side note, it's what I love about the name Katello: it's unique and there's no overlap with other existing names. Foreman has the problem that besides being a generic English word, it also has another Ruby project with the same name but serving a different goal. Worse, in development foreman.git actually lists the other foreman project as a development dependency.

Comment (melcorr) [reply _includes/manuals/nightly/1.2_release_notes.md]: Pulp is even worse. A nightmare actually. 
There's the British band and several software projects all competing on Google
https://twitter.com/pulp_platform/
https://pypi.org/project/PuLP/
The YouTube auto close captioning on one of the engineer's demos picked up Poop instead of Pulp. That would actually be easier and more unique software project name =)

**Comment by melcorr** [comment]
Actually there's no cherry-picking as there is only one branch. I'll raise another PR and just make the changes in the other directories.

**Comment by ekohl** [comment]
Oh yes, I should have been clearer in that. It's fairly manual here. What works is:
```console
$ git format-patch HEAD~2
0001-Standardize-capitalization-of-Foreman-Smart-Proxy.patch
0002-Fix-typo-in-Debian.patch
$ cd _includes/manuals/3.1
$ patch -p4 < ../../../0001-Standardize-capitalization-of-Foreman-Smart-Proxy.patch
patching file 1.1_architecture.md
```

Then do the `cd` and `patch` part for each manual version. Not as pretty as `git cherry-pick`, but it works reasonably well.

**Comment by melcorr** [comment]
> ```
> git format-patch HEAD~2
> ```

First time I am doing this so just to clarify: why the `~2` ?
We only want the architecture part for the other manuals, right? The EL8 statement is new to 3.2?

**Comment by upadhyeammit** [comment]
> > ```
> > git format-patch HEAD~2
> > ```
> 
> First time I am doing this so just to clarify: why the `~2` ? We only want the architecture part for the other manuals, right? The EL8 statement is new to 3.2?

~2 refers to top 2 commits of the branch. 

The EL7 statement is there on 3.1,
https://github.com/theforeman/theforeman.org/blob/gh-pages/_includes/manuals/3.1/1.2_release_notes.md#running-foreman-on-el7

And 3.1 has Ubuntu 18.04 note which also needs correction,

https://github.com/theforeman/theforeman.org/blob/gh-pages/_includes/manuals/3.1/1.2_release_notes.md#running-foreman-on-ubuntu-1804-dropped

**Comment by ekohl** [comment]
> First time I am doing this so just to clarify: why the `~2` ?

https://www.git-scm.com/docs/gitrevisions#Documentation/gitrevisions.txt-emltrevgtltngtemegemHEADmaster3em is the official documentation:

> A suffix ~ to a revision parameter means the first parent of that commit object. A suffix ~<n> to a revision parameter means the commit object that is the <n>th generation ancestor of the named commit object, following only the first parents. I.e. <rev>~3 is equivalent to <rev>^^^ which is equivalent to <rev>^1^1^1. See below for an illustration of the usage of this form.

So `HEAD` refers to the current commit. Formatting a patch of the changes since a current commit is asking for no changes. So you can specify `HEAD^` to get a patch for the latest commit. In this particular case it wasn't the latest commit, but the one before that. You can write that as `HEAD^^` or `HEAD~2`.

This revision specification is used all over git, so you can also use `git show HEAD^` to show the commit before the last. What also works is `git cherry-pick master^ master` to cherry pick the last 2 commits in master. This is equivalent to `git cherry-pick master^..master` So you can also expand this to `master~5..master` or `origin/master..master`. And if you've gotten this far, this also works with `git log`.

It may be a bit confusing that `format-patch` is the changes since the commit you specified where others are the exact changes, but the reasoning is that you can say something like `git format-patch origin/master` to create the patches in your branch compared to `origin/master`.

**Comment by melcorr** [comment]
Sorry - my question related to the fact that I purposely split the commit into two so we could patch just the architecture component of the guide. I understand what the ~2 did but I was wondering why. 
So there was no need to split this into two commits.

**Comment by ekohl** [comment]
No, it certainly helped it was 2 commits. The second commit did not need to be cherry picked, or at least different depending on versions.

**Comment by melcorr** [comment]
I don't think anyone got my point. For the record I use the `~2` `~` whatever daily. That wasn't the issue.

**Comment by ekohl** [comment]
With your later clarification I did get it. And I do think splitting it made sense. One patch cleanly applied everywhere and can safely be applied to other branches. The release notes changes were special and there I think manual application in older versions is easier than trying to use patches.

And you could have asked `git format-patch` to only generate a single patch but I thought it was easier to generate both files and ignore one.
